src/plugins/github/parseReferences.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+function findAllMatches(re: RegExp, s: string): any[] {
+  // modified from: https://stackoverflow.com/a/6323598
+  let m;
+  const matches = [];
+  do {
+    m = re.exec(s);
+    if (m) {
+      matches.push(m);
+    }
+  } while (m);
+  return matches;
+}
+
+export function findNumericReferences(body: string): number[] {
+  return findAllMatches(/(?:\W|^)#(\d+)(?:\W|$)/g, body).map((x) => +[x[1]]);
+}
+
+export type GithubExactUrlMatch = {|
+  +repoName: string,
+  +repoOwner: string,
+  +parentType: "pull" | "issues",
+  +number: number,
+  +commentFragment: ?{|
+    +fragmentType:
+      | "issue"
+      | "issuecomment"
```

Comment:
This also refers to a pull request comment, right? or not right? Fine with me if you want to keep these strings "opaque"/"straight-from-GitHub" (at least for now), but probably should document this one either way.

Reply:
Yeah, when pull requests get comments they moonlight as issues. added comments.